File: redwood-yoga-java/src/main/java/io/github/orioncraftmc/meditate/internal/enums/YGOverflow.java
package io.github.orioncraftmc.meditate.internal.enums;

public enum YGOverflow {
    YGOverflowVisible,
    YGOverflowHidden,
    YGOverflowScroll;

    public static final int SIZE = java.lang.Integer.SIZE;

    public static YGOverflow forValue(int value) {
        return values()[value];
    }

    public int getValue() {
        return this.ordinal();
    }
}

File: redwood-yoga/src/commonMain/kotlin/app/cash/redwood/yoga/internal/enums/YGOverflow.kt
package app.cash.redwood.yoga.internal.enums

enum class YGOverflow {
    YGOverflowVisible, YGOverflowHidden, YGOverflowScroll;

    fun getValue(): Int {
        return ordinal
    }

    companion object {
        fun forValue(value: Int): YGOverflow {
            return values()[value]
        }
    }
}
